cmd/createcluster.go

```diff
@@ -191,6 +194,47 @@ func runCreateCluster(w io.Writer, conf clusterConfig) error {
 		}
 	}
 
+	// TODO(xenowits): add flag to specify the number of distributed validators in a cluster
+	// Currently, we assume that we create a cluster of ONLY 1 Distributed Validator
+	var depositDatas [][]byte
+	for i := 0; i < conf.NumDVs+1; i++ { // Note that default value of conf.NumDVs is 0
```

Comment:
As suggested above remove this for loop and just create a deposit data for single DV.
And add a TODO to add deposit datas when the flag is introduced.